### PR TITLE
Fix/ui nodesjs version doc update 2.8.0

### PR DESCRIPTION
**Pull request by ajainarayanan**
Associated JIRA: https://issues.cask.co/browse/CDAP-1804

### CODE REVIEW

File: cdap-docs/integrations/source/partners/cloudera/configuring.rst [hunk 1]
```diff
@@ -41,7 +41,7 @@ the 'cdap' user installed by the parcel.
 Prerequisites
 =======================================
 
-#. Node.js (version 0.8.16 or greater) must be installed on the node(s) where the Web-App
+#. Node.js (version from 0.8.16 through 0.10.36) must be installed on the node(s) where the Web-App
    role instance will run. You can download the latest version of Node.js from `nodejs.org
```
Comment (awholegunch) [inline cdap-docs/integrations/source/partners/cloudera/configuring.rst]: We don't want the latest version of node. (!)
```
You can download an appropriate version of Node.js from `nodes.org <http://nodejs.org/dist/>`__.
```

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 2]
```diff
@@ -129,7 +129,7 @@ Node.js Runtime
 +++++++++++++++
 You can download the latest version of Node.js from `nodejs.org <http://nodejs.org>`__:
 
-#. The version of Node.js must be v0.8.16 or greater.
+#. The version of Node.js must be from v0.8.16 through v0.10.36.
 #. Download the appropriate Linux or Solaris binary ``.tar.gz`` from
    `nodejs.org/download/ <http://nodejs.org/download/>`__.
```
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: download will get the latest. We should suggest 
```
#. Download the appropriate Linux or Solaris binary ``.tar.gz`` for a version in the above range from
    `nodejs.org/dist/ <http://nodejs.org/dist/>`__.
```

File: cdap-docs/integrations/source/partners/cloudera/configuring.rst (hunk 1 above)
Comment (ajainarayanan) [inline cdap-docs/integrations/source/partners/cloudera/configuring.rst]: :smile: Good catch. Updated.

File: cdap-docs/integrations/source/partners/cloudera/configuring.rst [hunk 3]
```diff
@@ -41,8 +41,8 @@ the 'cdap' user installed by the parcel.
 Prerequisites
 =======================================
 
-#. Node.js (version 0.8.16 or greater) must be installed on the node(s) where the Web-App
-   role instance will run. You can download the latest version of Node.js from `nodejs.org
+#. Node.js (version from 0.8.16 through 0.10.36) must be installed on the node(s) where the Web-App
+   role instance will run. You can download the appropriate version of Node.js from `nodejs.org
    <http://nodejs.org>`__.
```
Comment (awholegunch) [inline cdap-docs/integrations/source/partners/cloudera/configuring.rst]: You should give them the dist link; it's really hard to find on their website:

http://nodejs.org/dist/

**Comment by awholegunch** [comment]
These changes need to be propagated to the other PRs (#1657, #1659).

**Comment by awholegunch** [comment]
LGTM.

**Pull request merged**